assets/src/design-system/components/textArea/index.js

```diff
@@ -39,6 +39,14 @@ const Container = styled.div`
   min-width: 100px;
 `;
 
+const CounterText = styled(Text).attrs({
+  size: THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL,
+  as: 'span',
+})`
+  color: ${({ theme }) => theme.colors.fg.tertiary};
+  font-size: 12px;
```

Comment:
For some reason, the `size: THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL` doesn't have an effect on the font-size here 🤷  It only takes effect correctly when using the `Text` directly, so defining the font-size explicitly here. Ideas?

Reply:
Magic! And it works, awesome. Thank you!